static/styles/left_sidebar.css

```diff
@@ -345,14 +345,12 @@ li.top_left_recent_topics {
 
 .conversation-partners,
 .topic-name {
-    /* stylelint-disable-next-line  value-no-vendor-prefix */
-    display: -webkit-box;
+    display: block;
     width: calc(100% - 5px);
+    white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
     padding-right: 2px;
-    -webkit-line-clamp: 2;
-    -webkit-box-orient: vertical;
```

Comment:
These changes appear to have been accidentally added to this PR; `git pull --rebase` should drop them for you.

Reply:
@timabbott  Yeah, i guess it solved the problem after your commit, would be great if can be merged.